src/redux/categories/categories.js

```diff
@@ -0,0 +1,24 @@
+const CHECK_STATUS = './library/categories/CHECK_STATUS';
+
+const stateInit = {
+  type: 'action',
+  checked: true,
+  payload: 'Under construction',
+};
```

Comment:
- [ ] According to the project requirement, your `initialState` should return an empty array. Kindly implement it. 👍